nuxeo-core/nuxeo-core-convert-plugins-test/src/test/java/org/nuxeo/ecm/core/convert/plugins/tests/advanced/AdvancedMSOfficeConverterTest.java

```diff
@@ -35,6 +35,8 @@
 
 public class AdvancedMSOfficeConverterTest extends SimpleConverterTest {
 
+    protected String expectedContentFilename;
```

Comment:
Seems never set, how can this work :thinking:

Reply:
It's set in the subclass — I didn't want to change the whole API of `SimpleConverterTest`

Reply:
OK, still sounds weird but fine.
Anyway, there is only one subclass of `AdvancedMSOfficeConverterTest`...